loops.js

```diff
@@ -50,4 +61,14 @@
 // >> 8
 // >> 9
 // ==========================================
-
+for (let i = 0; i < 10; i++) {
+    if (i < 3) {
+        console.log(i);
+    }
+    else if (i > 2 && i < 6) {
+        console.log("> " + i)
+    }
+    else {
+        console.log(">> " + i)
+    }
+}
```

Comment:
loops.js is helemaal goed, lekker bezig.

Reply:
Thanks!